scripts/gen_fh.py

```diff
@@ -86,7 +86,8 @@
 {{{{< friday_hack_header
     venue="{venue}"
     date="{month} {day}"
-    fb_event="#" >}}}}
+    food="pizza"
```

Comment:
can we really have pizza? (just my random thought, ignore this haha)

Reply:
I think we can but we need to confirm this with Pezzo again tmr once they open their shops